cmd/minishift/cmd/addon/addon.go

```diff
@@ -23,15 +23,15 @@ import (
 const (
 	addOnConfigKey = "addons"
 
-	emptyAddOnError = "You must specify an add-on name. Use `minishift addons list` to view installed add-ons."
+	emptyAddOnError = "You must specify an add-on name. Run `minishift addons list` to view installed add-ons."
 	noAddOnMessage  = "No add-on with the name %s is installed."
 )
 
 var AddonsCmd = &cobra.Command{
 	Use:     "addons SUBCOMMAND [flags]",
 	Aliases: []string{"addon"},
-	Short:   "Manages Minishift add-ons",
-	Long:    "Manages Minishift add-ons. You can install, list, enable or disable Minishift add-ons.",
+	Short:   "Manages Minishift add-ons using sub-commands.",
```

Comment:
IMO we do not need to mention sub-commands because the context is correct when we say  `Manages Minishift add-ons`. Anyway the user will see the sub-commands in the usage of `minishift addons --help`. 

Without this change the the short description shows up in `minishift` usage i.e. `minishift --help` which looks correct to me.
```
$ minishift 
Minishift is a command-line tool that provisions and manages single-node OpenShift clusters optimized for development workflows.

Usage:
  minishift [command]

Available Commands:
  addons      Manages Minishift add-ons
  config      Modifies Minishift configuration properties.
  console     Opens or displays the OpenShift Web Console URL.
  delete      Deletes the Minishift VM.
  docker-env  Sets Docker environment variables.
  help        Help about any command
  hostfolder  Manages host folders for use by the OpenShift cluster.
  ip          Gets the IP address of the running cluster.
  logs        Gets the logs of the running OpenShift cluster.
  oc-env      Sets path of 'oc' binary.
  openshift   Interact with an Openshift Cluster
  ssh         Log in to or run a command on a Minishift VM with SSH.
  start       Starts a local OpenShift cluster.
  status      Gets the status of the local OpenShift cluster.
  stop        Stops the running local OpenShift cluster.
  update      Update to latest version of Minishift.
  version     Gets the version of Minishift.

Flags:
      --alsologtostderr                  log to standard error as well as files
  -h, --help                             help for minishift
      --log_backtrace_at traceLocation   when logging hits line file:N, emit a stack trace (default :0)
      --log_dir string                   If non-empty, write log files in this directory (default "")
      --logtostderr                      log to standard error instead of files
      --show-libmachine-logs             Show logs from libmachine.
      --stderrthreshold severity         logs at or above this threshold go to stderr (default 2)
  -v, --v Level                          log level for V logs
      --vmodule moduleSpec               comma-separated list of pattern=N settings for file-filtered logging

Use "minishift [command] --help" for more information about a command.
```